pan3d/viewer/ui.py

```diff
@@ -16,204 +16,187 @@ def initialize(server):
-                                        v_model=("grid_z_array", None),
-                                        items=("coordinates",),
-                                        hide_details=True,
-                                        dense=True,
-                                        style="max-width: 250px;",
-                                    )
-                                    with vuetify.VBtn(
-                                        icon=True,
-                                        click="grid_z_array = undefined",
-                                    ):
-                                        vuetify.VIcon("mdi-delete")
-                                with vuetify.VRow():
-                                    html.Div("T:", classes="text-subtitle-2 pr-2")
-                                    html.Div("{{ grid_t_array || 'Undefined' }}")
-                                    vuetify.VSpacer()
-                                    vuetify.VSlider(
-                                        label="Time",
-                                        v_show="grid_t_array && time_max > 0",
-                                        v_model=("time_index", 0),
-                                        min=0,
-                                        max=("time_max", 0),
-                                        step=1,
-                                        dense=True,
-                                        hide_details=True,
-                                        style="max-width: 250px;",
-                                    )
-                                    vuetify.VSelect(
-                                        v_model=("grid_t_array", None),
-                                        items=("coordinates",),
-                                        hide_details=True,
-                                        dense=True,
-                                        style="max-width: 250px;",
-                                    )
-                                    with vuetify.VBtn(
-                                        icon=True,
-                                        click="grid_t_array = undefined",
-                                    ):
-                                        vuetify.VIcon("mdi-delete")
-                    with html.Div(
-                        style="display: flex; flex: 1; height: calc(100vh - 300px);"
-                    ):
-                        with vtk.VtkRemoteView(
-                            ctrl.get_render_window(),
-                            # v_show="view_mode === 'view_grid'",
-                            interactive_ratio=1,
-                        ) as vtk_view:
-                            ctrl.view_update = vtk_view.update
-                            ctrl.reset_camera = vtk_view.reset_camera
+                    with vtk.VtkRemoteView(
+                        ctrl.get_render_window(),
+                        interactive_ratio=1,
+                    ) as vtk_view:
```

Comment:
You may want to use PyVista's views: https://github.com/pyvista/pyvista/blob/main/pyvista/trame/views.py

- `from pyvista.trame.views import PyVistaRemoteView`
- `from pyvista.trame.ui import plotter_ui`

Reply:
Thanks! Changed in [08db4f5](https://github.com/Kitware/pan3d/pull/23/commits/08db4f5c93448598b8bf9e360f7f2b2ec60e083a)

Reply:
It looks like `plotter_ui` returns a `PyVistaRemoteView` object, so I only imported and directly used `plotter_ui`.